### PR TITLE
sdflkj

### PR DESCRIPTION
**Pull request by lfundaro**
lksjdfslfkj

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,7 @@
 test repo
 
 more text
+
```
Comment (lfundarotest) [inline README.md]: we need to change this line

File: README.md [hunk 2]
```diff
@@ -1,3 +1,7 @@
...
+
+
```
Comment (lfundarotest) [inline README.md]: want to add additional things here.

File: README.md [hunk 3]
```diff
@@ -1,3 +1,7 @@
 test repo
```
Comment (lfundarotest) [inline README.md]: onethinkg here

File: README.md [hunk 4]
```diff
@@ -1,3 +1,7 @@
...
 
 more text
+
+
```
Comment (lfundarotest) [inline README.md]: another one here.

File: README.md [hunk 5]
```diff
@@ -1,3 +1,7 @@
 test repo
 
 more text
```
Comment (lfundarotest) [inline README.md]: why

Comment (lfundaro) [reply README.md]: why not ?

Comment (lfundaro) [reply README.md]: why why ?

File: README.md (hunk 1 above)
Comment (lfundarotest) [inline README.md]: fjfdslk

**Pull request merged**